styles.css

```diff
@@ -65,6 +83,248 @@ nav li a img {
   background-color: white;
 }
 
+@media screen and (min-width: 768px) {
+  body {
+    width: 100%;
+  }
+
+  .headline {
+    display: flex;
+    flex-wrap: wrap;
+    background-image: url('desktopbackground1.svg');
+    background-repeat: no-repeat;
+    background-size: cover;
+    border-bottom-left-radius: 50px;
+    padding: 30px;
+    width: 100%;
+    height: 779px;
+  }
+
+  .textdiv {
+    position: relative;
+    left: 140px;
+    top: 180px;
+    width: 720px;
+    height: 98px;
+  }
+
+  .leftli {
+    display: none;
+  }
+
+  .link-desktop {
+    display: inline-block;
+  }
+
+  .textdiv h1 {
+    font-size: 48px !important;
+    line-height: 60px;
+  }
+
+  .textdiv p {
+    font-size: 20px !important;
+    line-height: 28px;
+  }
+
+  .container {
+    display: none !important;
+  }
+
+  .containerdesktop {
+    display: flex !important;
+    flex-direction: row;
+  }
+
+  .tonicdesktop {
+    flex: 50%;
+    max-width: 45%;
+  }
+
+  .imgdivdesktop {
+    flex: 50%;
+    max-width: 45%;
+  }
+
+  .popheaddesktop {
+    font-weight: 700;
+    font-size: 40px !important;
+    line-height: 52px;
+    color: #091e42 !important;
+  }
+
+  .pop13desktop {
+    font-weight: 500 !important;
+    font-size: 18px !important;
+    line-height: 24px;
+  }
+
+  .pop15desktop {
+    font-size: 16px !important;
+  }
+
+  .container2 {
+    display: none !important;
+  }
+
+  .containerdesktop2 {
+    display: flex !important;
+    flex-direction: row;
+  }
```

Comment:
I am able to understand your point regarding !important but for now in this project please allow me as an exception because some of my properties are not changing in Media queries without using important that's why it become necessary for me to use important. Please allow as an exception in this one. Thanks